### ISSUE TITLE
Programmatically set UISelect selected option

### ISSUE THREAD

**Issue by lowerends** [JosephusPaye/Keen-UI#77]
Hi,

Is there a way to programmatically set the selected option in a UISelect component? The value property itself can be updated using two-way binding, but the selected option does not change with it.

Thanks

**Comment by victor-ponamariov** [comment]
In addition: is there any example of how to set selected (default) value for UiSelect? I'm struggling with that, it's not a problem to fill select with options, but I didn't manage to set some of the options by default =\

**Comment by EmilMoe** [comment]
You need to have reference to the same array that contains the values.

~~~~
<ui-select name=color label="Favourite color" :options=colors placeholder="Select a color" :default=colors[0]></ui-select>
~~~~

**Comment by JosephusPaye** [comment]
Implemented in https://github.com/JosephusPaye/Keen-UI/commit/c27e935d848d983bf6a1877593cc9e4ccc4b5cb2

**Comment by JosephusPaye** [comment]
Now available in [v0.8.9](https://github.com/JosephusPaye/Keen-UI/releases/tag/v0.8.9) with the new `ui-select::set-selected` event on [UiSelect](https://josephuspaye.github.io/Keen-UI/#/ui-select-docs).

**Comment by JosephusPaye** [comment]
In v1.0, you can simply update the `value` prop and the selected option will be updated accordingly.
